README.md

```diff
@@ -23,30 +23,33 @@ In IEEE Workshop on Applications of Signal Processing to Audio and Acoustics (WA
 ## Installation
 
 ### Non-python dependencies
-Scaper has two non-python dependencies:
-- SoX: http://sox.sourceforge.net/
+Scaper has one non-python dependency:
 - FFmpeg: https://ffmpeg.org/
 
+If you are installing Scaper on Windows, you will also need:
+- SoX: http://sox.sourceforge.net/
+
+If you are installing on Linux/macOS, the SoX dependency is taken care of via 
+[SoxBindings](https://github.com/pseeth/soxbindings).
```

Comment:
"On Linux/macOS SoX is replaced by [SoxBindings](https://github.com/pseeth/soxbindings) which is significantly faster, giving better overall performance in Scaper. On these platforms SoxBindings is installed automatically when calling `pip install scaper` (see below)."